src/platform/imx93_a55/lib/dai.c

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright 2023 NXP
+ */
+
+#include <rtos/sof.h>
+#include <sof/drivers/sai.h>
+#include <sof/drivers/edma.h>
+#include <sof/lib/memory.h>
+
+static struct dai sai[] = {
+	{
+		.index = 3, /* SOF uses SAI3 */
```

Comment:
This index is slightly weird -- other i.MX platforms just use indices 0 and 1 despite it being like ESAI1 and SAI2 or similar.

You _can_ set it as 3 if you believe this is better. What's important is this index must match a value in the topology (so once this is merged you cannot really ever change it).

Reply:
The only reason I set it to 3 is because at the time I believed it needs to match the SAI index which, according to what you just said, is not the case. I'd say leave this as-if for now as it makes more sense (at least to me) for it to match the SAI index. There's no problem with the topology so it should be fine.